website/theses/templates/theses/conversion.html

```diff
@@ -10,15 +10,26 @@
 
 
 {% block content %}
-    <h1>
-        Thank you for your interest!
-    </h1>
+    <h1>Thank you for your interest!</h1>
     <a href="https://www.facebook.com/share.php?u={{ origin_url }}&title=My new thesis topic! {{ title }}">
         <h3>
             Share this great news with your friends on Facebook!
         </h3>
     </a>
 
+
+    <!-- Facebook Share Button -->
+    <div id="fb-root"></div>
+    <script>(function(d, s, id) {
```

Comment:
Instead of having this in the main `content` block, which is rendered in `body` from the inherited `theses_base.html` (as can be seen in the first line of this template), it would be better to add it to `extra_js_2` block, which would render this in a common `script` part of the page.

It can be seen here: https://github.com/ea-czech-republic/efektivnialtruismus.cz/blob/redesign/website/theses/templates/theses/theses_base.html#L204 . 

So could you please move it into:

```
{% block extra_js_2 %}
 ... the script code here ...
{% endblock %}
```

is it understandable how it works? It's utilizing django's template inheritance and template blocks/tags. I can explain on a call if you want.